src/platform/cannonlake/include/platform/interrupt.h

```diff
@@ -85,7 +85,7 @@
 /* IRQ Level 5 bits */
 #define IRQ_BIT_LVL5_LP_GP_DMA1	15
 #define IRQ_BIT_LVL5_LP_GP_DMA0	16
-#define IRQ_BIT_LVL5_DMIC		6
+#define IRQ_BIT_LVL5_DMIC(x)		(6 + x)
```

Comment:
I check the CNL and APL spec, on APL, DMIC interrupt is at pin 6, but on CNL it should be 8.
please conform it.

Reply:
Yep, good catch!